legal-api/src/legal_api/core/filing.py

```diff
@@ -207,7 +208,7 @@ def get_json(self) -> Optional[Dict]:
             filing_json = filing
 
         else:  # Filing.Status.COMPLETED.value
-            filing_json = VersionedBusinessDetailsService.get_revision(self.id, self._storage.legal_entity_id)
+            filing_json = VersionedBusinessDetailsService.get_revision(self.id, self._business)
```

Comment:
@argush3 I don't think we are using `get_json` function from core filing anymore. Will remove this later